REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.3
+julia 0.4
```

Comment:
It's updated
On Thu, Jun 30, 2016 at 12:49 PM Steven G. Johnson <notifications@github.com>
wrote:

> In REQUIRE
> <https://github.com/stevengj/LaTeXStrings.jl/pull/9#discussion_r69168264>:
>
> > @@ -1,2 +1,2 @@
> > -julia 0.3
> > +julia 0.4
>
> You also need to update .travis.yml
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/stevengj/LaTeXStrings.jl/pull/9/files/24a315f6f0c68e77e4612b097db20c42b17ed311..7e03e2a817a3df3a66710251487473e082886d0d#r69168264>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA8SvQvPDC_shu_FvqQvMEgtZkoz_GD_ks5qQ_N8gaJpZM4JCPNN>
> .
>